components/layouts/PopupLayout.tsx

```diff
@@ -16,7 +16,7 @@ export default function PopupLayout() {
 
   return (
     <PostHogWrapperProvider>
-      <div className="h-[600px] w-[375px] bg-icy-blue-950 font-sans text-white">
+      <div className="h-[600px] w-[375px] overflow-y-scroll bg-icy-blue-950 font-sans text-white">
```

Comment:
I think we should make not scrollable in the production mode.

![image](https://github.com/user-attachments/assets/84c5c050-9bf7-419b-85be-ee0069ea4b86)

Reply:
9798ab58f9ca862d8be0aa7536cc9ada5a03a926

In most of our screens, it's pretty obvious that you can scroll. So I just hid it. Let me know what you think.

Reply:
Great to hide it to me.